README.adoc

```diff
@@ -58,7 +58,7 @@ In addition, you can review the release notes and project roadmap here on GitHub
 [[community]]
 == Community
 
-Minishift is an open-source project dedicated to developing and supporting Minishift.
+Minishift is an open-source project dedicated to developing and supporting OpenShift.
```

Comment:
Lets remove this line. and move the below line  `The code base is forked from the https://github.com/kubernetes/minikube[Minikube] project.` as the last sentence of https://github.com/minishift/minishift#welcome-to-minishift